example/distill/nlp/README.md

```diff
@@ -0,0 +1,50 @@
+# ERNIE distillation
+
+## Dataset
+Chinese sentiment dataset
+
+## Local test
+## Finetune or downlod finetuned model
+### Download dataset
+```
+wget https://paddle-edl.bj.bcebos.com/distillation/chnsenticorp/data.tgz
+tar -xzvf data.tgz
+```
+
+### Finetune
```

Comment:
finetuning is against pretraining. In distillation, it is the `teacher model` not the `finetuned model`.

Reply:
Done.